CONTRIBUTING.rst

```diff
@@ -13,31 +13,24 @@ First of all, clone the repository::
 
     $ git clone git@github.com:aio-libs/aiobotocore.git
 
-Create virtualenv with at least python3.8 (older versions are not supported).
-For example, using ``virtualenvwrapper`` commands could look like::
-
-   $ cd aiobotocore
-   $ mkvirtualenv --python=`which python3.8` aiobotocore
+Make sure the Python package and project manager `uv <https://docs.astral.sh/uv/>`_ is installed.
 
+Create a virtual environment::
 
-After that, please install libraries required for development::
-
-    $ pip install pip-tools
-    $ pip-compile --all-extras pyproject.toml requirements-dev.in
-    $ pip-sync
-    $ pip install -e .
+   $ cd aiobotocore
```

Comment:
This probably shouldn't be dedented. Does it render well?

Reply:
It was rendering well (https://aiobotocore--1257.org.readthedocs.build/en/1257/contributing.html), but thanks for pointing out the mixed up indentations. It's fixed now.